### ISSUE TITLE
The training loss is extramely large (close to 3.368e6)?

### ISSUE THREAD

**Issue by Songlin1998** [RameenAbdal/StyleFlow#130]
Hi! Thank you for your contribution. This is an excellent work.

I try to train my model, but the training loss is extramely large, close to 3.368e6.

Is this right？After several epochs of training, the reconstrution result is not good.
